src/plugins/intel_cpu/tests/unit/shape_inference_test/custom_shape_infer/binary_elementwise_arithmetic.cpp

```diff
@@ -83,7 +83,11 @@ TYPED_TEST_P(CpuShapeInferenceTest_BEA, shape_inference_aubtob_none) {
 }
 
 TYPED_TEST_P(CpuShapeInferenceTest_BEA, shape_inference_aubtob_none_incompatible_shapes) {
-    GTEST_SKIP() << "Skipping test, please check CVS-108946";
+    GTEST_SKIP() << "Skipping test, eltwise only implemented numpy type boardcast";
```

Comment:
Eltwise shape infer most definitely doesn't support broadcasting types other then numpy. But this may be resolved through switching to the ngraph shape inference when other broadcasting types are specified. This can be done in the shape inference factory. Another question is, whether the Node itself support different broadcasting.

Reply:
when I added code  to do shape infer for different broadcast property , I found that op lost the broadcast property when converted to cpu graph.
1. sometimes op is converted to powerstatic, but powerstatic lost the broadcast property.
2. when op fuse, there is some issue, for example  Tanh fuse with Add op, the Tanh's broadcast type is null, and
    Add's broadcast type is numpy, so the broadcast type should be numpy, but if check the broadcast type of Tanh, it' s null
I will look deep into it and check how to fix it

Reply:
Ok then, probably we can make a ticket and skip the tests for now.

Reply:
already add a ticket for it and skipped the test case